components/script/dom/htmltablecellelement.rs

```diff
@@ -123,6 +128,20 @@ impl HTMLTableCellElementLayoutHelpers for LayoutJS<HTMLTableCellElement> {
                 .unwrap_or(LengthOrPercentageOrAuto::Auto)
         }
     }
+
+    fn get_inherited_cellpadding(&self) -> Option<u32>
+    {
+        unsafe {
+            // Follow chain: Cell -> Row -> Section -> Table
+            self.upcast::<Node>().parent_node_ref()
```

Comment:
Hmm... This works on a well-formed table, but what should happen if, for example, the row is anonymous? What do other browsers / the spec do?